modules/nw-route-specific-annotations.adoc

```diff
@@ -29,6 +29,7 @@ and "-". The default is the hashed internal key name for the route. |
 |`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`haproxy.router.openshift.io/ip_whitelist` | Sets a whitelist for the route. |
 |`haproxy.router.openshift.io/hsts_header` | Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. |
+|`haproxy.router.openshift.io/log-send-hostname` | Sets the hostname field in the Syslog header. Uses the hostname of the system. `log-send-hostname` is enabled by default if any Ingress API logging method, such as sidecar or Syslog facility, is enabled for the router. |
```

Comment:
* The first "hostname" should be in backticks if it's an actual field.
* The second "hostname" instance should be two words.

Reply:
Thank you!